src/main/java/horseshoe/TemplateLoader.java

```diff
@@ -577,11 +608,11 @@ private TagRenderer processTag(final Stack<Loader> loaders, final TemplateLoadSt
 				if (tag.length() > 1 && tag.charAt(1) == '>') { // Section Partial
 					final TemplateRenderer partialRenderer = loadPartial(loaders, state, tag.substring(1));
 
-					if (partialRenderer.getTemplate() == null) {
+					if (partialRenderer.getTemplate() == null && !partialRenderer.deferred()) {
```

Comment:
I'm thinking something like
```java
final Template partial;

if (partialRenderer.deferred()) {
    partial = new Template(null, state.toLocation());
} else if (partialRenderer.getTemplate() == null) {
    throw ...
} else {
    partial = new Template(partialRenderer.getTemplate().getSection().getName(), state.toLocation());
}
```
might be a little bit cleaner. Thoughts?

Reply:
Reverted to original